features/waterOrdering/index.js

```diff
@@ -78,13 +57,36 @@ const passedEnoughTimeFromLastOrder = async (robot) => {
   const lastWaterOrder = await lastWaterOrderCreatedAt(robot)
 
   if (lastWaterOrder) {
-    const currentDate = new Date();
+    const currentDate = new Date()
 
-    if ((currentDate - lastWaterOrder) / 1000 >
-      (process.env.WATER_ORDER_MIN_INTERVAL || DEFAULT_WATER_ORDER_INTERVAL)) {
-      return (true);
+    if ((currentDate - lastWaterOrder) / 1000
+      > (process.env.WATER_ORDER_MIN_INTERVAL || DEFAULT_WATER_ORDER_INTERVAL)) {
```

Comment:
can this be split into several functions for better readability?

Reply:
@AleksSenkou yeah, it is just a start of refactoring. Created PR so early to check if our Github  Actions is working